plugins/generator-1.17.1/forge-1.17.1/templates/particle.java.ftl

```diff
@@ -0,0 +1,137 @@
+<#--
+ # MCreator (https://mcreator.net/)
+ # Copyright (C) 2020 Pylo and contributors
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # Additional permission for code generator templates (*.ftl files)
+ #
+ # As a special exception, you may create a larger work that contains part or
+ # all of the MCreator code generator templates (*.ftl files) and distribute
+ # that work under terms of your choice, so long as that work isn't itself a
+ # template for code generation. Alternatively, if you modify or redistribute
+ # the template itself, you may (at your option) remove this special exception,
+ # which will cause the template and the resulting code generator output files
+ # to be licensed under the GNU General Public License without this special
+ # exception.
+-->
+
+<#-- @formatter:off -->
+<#include "procedures.java.ftl">
+
+package ${package}.particles;

```

Comment:
You should also change the package for `client.particles`, so it follows Vanilla.

Reply:
When I try to replace `${package}.particle` with another package, particle MEs import themselves for some reason:
```gradle
C:\Users\user\AppData\Local\Temp\mcreator_test_workspace4162064558056833974\src\main\java\net\mcreator\test_mod\particle\Exampleparticle1Particle.java:21: error: package net.mcreator.test_mod.particle does not exist
import net.mcreator.test_mod.particle.Exampleparticle1Particle;
                                     ^
```